Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -1373,7 +1373,7 @@ private static (IStore Store, IStateStore StateStore) Loader(Uri storeUri)
             int txEpochUnitSeconds = query.GetInt32("tx-epoch-unit-secs", 86400);
             int blockEpochUnitSeconds = query.GetInt32("block-epoch-unit-secs", 86400);
             int dbConnectionCacheSize = query.GetInt32("connection-cache", 100);
-            bool @readonly = query.GetBoolean("readonly", false);
+            var parse = Enum.TryParse(query.Get("instance-type"), out RocksDbInstanceType type);
```

Comment:
How about using like this?

```suggestion
            var instanceType = Enum.TryParse(query.Get("instance-type"), out RocksDbInstanceType type) ? type : RocksDbInstanceType.Primary;
```